docs/developer-guide/mapstore-migration-guide.md

```diff
@@ -168,6 +240,7 @@ Contents of your `pluginsConfig.json` need to be reviewed to allow usage of new
 ```
 
 ### Updating extensions
+
```

Comment:
please, check if your ide applys some formatting, or it was you ?

Reply:
headers without a space below not correct in markdown. Depending on the interpreter (e.g. mkdocs / github ...) these may fail. I corrected many of them during the time that was producing errors in mkdocs (typically with the unsorted lists, that was not correctly recognized without a space). I have an extension in my IDE that notifies me the errors. We may add a spell check also in mkdocs to avoid future errors, anyway everytime I found something incorrect, I fix it.

Reply:
markdownlint for VSCode